factory/base.py

```diff
@@ -696,10 +696,7 @@ def _build(cls, model_class, *args, **kwargs):
             raise ValueError(
                 "ListFactory %r does not support Meta.inline_args." % cls)
 
-        # When support for Python <3.6 is dropped sorting will no longer be required
-        # because dictionaries will already be ordered, this can then be changed to:
-        # values = kwargs.values()
-        values = [v for k, v in sorted(kwargs.items(), key=lambda item: int(item[0]))]
+        values = kwargs.values()
```

Comment:
Maybe we should keep a comment reminding ourselves that this is sorted anyway through builtins?

Reply:
I added a comment, please let me know what you think.